sources/api/schnauzer/src/helpers.rs

```diff
@@ -1368,6 +1377,129 @@ pub fn etc_hosts_entries(
     Ok(())
 }
 
+/// This helper negates a boolean value, or returns a default value when the provided key wasn't
+/// set.
+///
+/// The first argument for the helper is the default value; the second argument is the key to
+/// negate. Both values must be booleans, otherwise the helper will return an error. The default
+/// value will be returned as it is if the provided key is missing.
+
+pub fn negate_or_else(
+    helper: &Helper<'_, '_>,
+    _: &Handlebars,
+    _: &Context,
+    renderctx: &mut RenderContext<'_, '_>,
+    out: &mut dyn Output,
+) -> Result<(), RenderError> {
+    // To give context to our errors, get the template name, if available.
+    trace!("Starting negate_or_else helper");
+    let template_name = template_name(renderctx);
+    trace!("Template name: {}", &template_name);
+
+    // Check number of parameters, must be exactly two (the value to negate and the default value)
+    trace!("Number of params: {}", helper.params().len());
+    check_param_count(helper, template_name, 2)?;
+
+    let default_value = get_param(helper, 0)?;
+    let value_to_negate = get_param(helper, 1)?;
+
+    let default = match default_value {
```

Comment:
GitHub is highlighting `default` as keyword - maybe pick a different identifier.